packages/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/cloudfirestore/CloudFirestorePlugin.java

```diff
@@ -36,9 +37,11 @@
 
   // Handles are ints used as indexes into the sparse array of active observers
   private int nextHandle = 0;
```

Comment:
Can you please rename this nextListenerHandle to better distinguish it from the property below?